dbflow/src/main/java/com/xmartlabs/base/dbflow/controller/DbFlowController.java

```diff
@@ -119,4 +119,11 @@ public Completable deleteEntity(@NonNull D entity) {
     return Completable.fromAction(entity::delete)
         .subscribeOn(Schedulers.io());
   }
+
+  /**
+   * Retrieves the type of the AppDataBase class
```

Comment:
shouldn't bee something like: `Retrieves the type of the AppDataBase model class`